src/kernel/sched/sched.c

```diff
@@ -469,7 +467,20 @@ static void __schedule(int preempt) {
 		spin_unlock(&rq.lock);
 
 		schedee_set_current(next);
-		log_debug("prev: %#x, next: %#x", prev, next);
+		
```

Comment:
Please extract modifications in 'sched.c' to separate PR. This part is very important and must change very carefully.